### ISSUE TITLE
Cellfie Import of Individuals with a GND-Identifier – "Missing prefix required"

### ISSUE THREAD

**Issue by sarahondraszek** [protegeproject/cellfie-plugin#163]
Dear Cellfie users,

I am currently trying to use Cellfie to import people with GND-identifiers into an ontology of a letter correspondence that I modelled.

To do so, I created some simple import rules in Cellfie. Unfortunately, I can not seem to find a solution to the problem that Cellfie and hence Protégé fail to recognise URLs that I want to use as resource identifiers. Whenever I try to import URLs from a cell with a set prefix (like "gnd" with the namespace http://d-nb.info/gnd/), I get an error **Missing prefix required** and when I try to apply some solution that I found online (like adding the prefix to the _Active Ontology_ tab or using **(mm:capturing("([^\/]+$)") mm:prefix="gnd")**) in the import rules, it still does not work.

As a result, the individuals are not imported as, for example., [https://d-nb.info/gnd/116765968](https://d-nb.info/gnd/116765968) but as _http://www.semanticweb.org/sarahreb/ontologies/2022/8/constance-de-salm#2027787-8_
(IRI of my ontology). Is there any solution to this problem? With geonames-identifiers and _mm:prefix="geonames"_, it works fine, but with other prefixes that I define in my ontology, I can not seem to make it work.

<img width="1196" alt="cellfie_rules" src="https://user-images.githubusercontent.com/69308007/193789208-f501b624-ca48-4d18-b867-af16f9a2fc7e.png">
<img width="308" alt="Capture d’écran 2022-10-04 à 11 50 33" src="https://user-images.githubusercontent.com/69308007/193789752-e45227d8-4df6-4751-9700-d5638bca7ba2.png">
Short remark: I tried using https and http, if that could solve the issue. That is why there are two spreadsheet versions with different URLs.

Thank you in advance for your help!

**Comment by martinjoconnor** [comment]
Can you post an OWL file, a spreadsheet, and a set of rules here that replicate the issue and I will take a look.

**Comment by sarahondraszek** [comment]
Of course, thank you a lot! I converted the OWL and rule-JSON file to TXT to upload them, I hope it works.
[filtered_cds_data.xlsx](https://github.com/protegeproject/cellfie-plugin/files/9713275/filtered_cds_data.xlsx)
[transformation_rules-cellfie.txt](https://github.com/protegeproject/cellfie-plugin/files/9713286/transformation_rules-cellfie.txt)
[cds-ontology.txt](https://github.com/protegeproject/cellfie-plugin/files/9713291/cds-ontology.txt)

**Comment by sarahondraszek** [comment]
Also some more screenshots that show how after the import of the GND-URLs, the namespace is changed to my local IRI but with geonames, the same import shows the actual geonames namespace.
<img width="646" alt="Capture d’écran 2022-10-05 à 08 39 28" src="https://user-images.githubusercontent.com/69308007/193996973-16b1209d-66c1-4e55-8c6b-ded58cb4fd71.png">
<img width="646" alt="Capture d’écran 2022-10-05 à 08 39 48" src="https://user-images.githubusercontent.com/69308007/193996982-088d5e7c-906a-4f79-938a-51294514f579.png">
<img width="646" alt="Capture d’écran 2022-10-05 à 08 40 29" src="https://user-images.githubusercontent.com/69308007/193996984-bb76b165-6813-4524-ab82-ebbf9ac19c98.png">
